public/components/datasources/components/manage/associated_objects/associated_objects_details_flyout.tsx

```diff
@@ -223,12 +225,17 @@ export const AssociatedObjectsDetailsFlyout = ({
 
   useEffect(() => {
     if (loadStatus.toLowerCase() === DirectQueryLoadingStatus.SUCCESS) {
```

Comment:
Let's handle the failed case where to stop loading the table.

Reply:
Loading is handled by `loadStatus`. If it fails, it'll be handled already.